backend/lightning/lightning.go

```diff
@@ -154,13 +167,51 @@ func (lightning *Lightning) Deactivate() error {
 	return nil
 }
 
+// AssertActive returns an error if the lightning service not has been activated.
+func (lightning *Lightning) AssertActive() error {
+	lightningConfig := lightning.backendConfig.LightningConfig()
+	if len(lightningConfig.Accounts) == 0 || lightning.sdkService == nil {
+		return errp.New("Lightning not initialized")
+	}
+	return nil
+}
+
+// Balance returns the balance of the lightning account, together with its conversions to fiat.
+func (lightning *Lightning) Balance() (interface{}, error) {
```

Comment:
don't return `interface{}` but a concrete type

Reply:
this function should really return `*accounts.Balance`, like btc/account.go and eth/account.go, and the handler should convert to `AccountBalance`.

Reason: AccountBalance is frontend focused, and it's the handler's job to convert input/output from/to the frontend.

Reply:
Agree :+1: done.

Reply:
Only now I see that the handlers are methods on the same struct :O but the split was still good, as we will hopefully refactor this later and make a clean split :smile: